src/main/js/helpers/constants.js

```diff
@@ -149,7 +149,7 @@ export const AXES_ORIENTATION = {
 export default {
     PADDING: {
         top: 10,
-        bottom: 5,
+        bottom: 2.5,
```

Comment:
Why would reducing the bottom padding fix this?

Reply:
So, bottom padding pushes the graph from the bottom towards the top. Hence by reducing it, we stop pushing the graph towards the top. And the previous solution suggested by @ShettyAkarsh i.e `(config.padding.top) * 2 + config.padding.bottom;` does the same as reducing the bottom padding by half. And this won't even affect the consistency between the graphs.